packages/config/config/devices/0x0460/qnsw-001P16.json

```diff
@@ -70,15 +74,16 @@
 		{
 			"#": "20",
 			"label": "O1: Auto-On Timer",
+			"description": "The timer resets to zero each time the Device receives an OFF command",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 32535,
 			"defaultValue": 0,
 			"unsigned": true
 		},
 		{
-			"#": "25",
-			"label": "O1: Auto-On/Off Timer Unit",
```

Comment:
I switched 23 and 25 on purpose because it IMO makes sense to have the unit right next to the timers.

Reply:
understood